static/app/views/dashboards/widgetCard/widgetCardDataLoader.tsx

```diff
@@ -87,8 +87,6 @@ export function WidgetCardDataLoader({
   if (widget.widgetType === WidgetType.RELEASE) {
     return (
       <ReleaseWidgetQueries
-        api={api}
-        organization={organization}
         widget={widget}
         selection={selection}
         limit={tableItemLimit}
```

Comment:
Regarding Line 94, is `dashboardFilters` global filters applied to the entire dashboard? Or is it specific to the widget. If it's the former it might be a good contender for a hook instead of a prop.